app/javascript/components/timeline-options/timeline-helper.js

```diff
@@ -90,6 +90,13 @@ export const buildChartDataObject = (rawData) => {
       datasets.push(obj);
     }
   });
+  datasets.sort((a, b) => {
+    const keyA = (a.date);
+    const keyB = (b.date);
+    if (keyA < keyB) return -1;
+    if (keyA > keyB) return 1;
+    return 0;
+  });
```

Comment:
Hey @MelsHyrule , I am not so sure of this, but will the below code works in this case?

```
datasets.sort((a, b) => new Date(b.date) - new Date(a.date));
```
This will return -1 or 0 or 1

Reply:
good catch